README.md

```diff
@@ -1,7 +1,8 @@
-# Ice Window Manager (IceWM)
+# Ice Window Manager (IceWM) Research for fix bug : uncorrectly size screen when we using multimonitor and xrandr
```

Comment:
Sorry, what do you mean with switch? --preferred does not switch anything, see manpage.

Then, please do a proper bug report and explain what you were expecting and what you actually see!

That said, I tried to reproduce and see weird effects. After switching and moving stuff around with arandr, icewm suddently creates all new windows with zero pixel size. Is this the issue you observed?